packages/components/segmented/src/segmented.vue

```diff
@@ -146,7 +146,7 @@ const selectedStyle = computed(() => ({
 
 const selectedCls = computed(() => [
   ns.e('item-selected'),
-  ns.is('disabled', state.disabled),
+  ns.is('disabled', getDisabled(getOption(props.modelValue))),
```

Comment:
If it has already been modified like this, does it mean the `state.disabled` is redundant code?

Reply:
@tolking yeah, sorry I didn't notice that before, I have deleted redundant code.

Reply:
@kooriookami And, please also help us see if there are any special circumstances that we have not considered?